### PR TITLE
Ongoing pylint improvements

### PR DESCRIPTION
**Pull request by SenlisO**
This pull request represents ongoing pylint improvements.  I'll let you know when a larger block of code is ready for the pull request to be accepted, but it can be pulled in at any time.

### CODE REVIEW

File: usr/share/system-installer/UI/main.py [hunk 1]
```diff
@@ -904,7 +904,9 @@ def make_space_parts(self, widget):
                 if "children" in each:
                     for each1 in each["children"]:
                         self.parts.append(each1["name"],
-                                          f"{each1['name']}, filesystem: {each1['fstype']}, size: {int(auto_partitioner.bytes_to_gb(each1['size']))}GB")
+                                          f"{each1['name']}, filesystem: "\
+                                            "{each1['fstype']}, size: "\
+                                            "{int(auto_partitioner.bytes_to_gb(each1['size']))}GB")
```
Comment (Batcastle) [inline usr/share/system-installer/UI/main.py:909]: Does this block of code actually work? If so and pylint doesn't complain then that's fine. Just looks more like shell code than Python.

**Review by Batcastle — COMMENTED** [review]
Just a quick question regarding a block of unusual looking code.

**Review by Batcastle — CHANGES_REQUESTED** [review]
A lot of these functions have extra arguments because GTK passes the widget that called the function to the function itself. Without that extra argument it can throw errors, crashing `system-installer`. Please be sure, when working in classes in UI files, that arguments for functions are not needed during run time.

File: usr/share/system-installer/UI/main.py (hunk 1 above)
Comment (SenlisO) [reply usr/share/system-installer/UI/main.py:911]: When I looked up how to split strings to multiple lines, this was the protocol.  Also, it made pylint happy.  I still need to test it though.

**Comment by SenlisO** [comment]
> A lot of these functions have extra arguments because GTK passes the widget that called the function to the function itself. Without that extra argument it can throw errors, crashing `system-installer`. Please be sure, when working in classes in UI files, that arguments for functions are not needed during run time.

I take it you are referring to the functions that include things like "button" as a param but it isn't used anywhere in the function itself.  In that case, I can add all of those back, add a flag so that pylint ignores it, and then put in docstrings that explain that those params belong there.

**Comment by SenlisO** [comment]
I added the params back and added a flag and a comment to ignore unused-params for those functions that need them for GTK

**Pull request closed without merging**